### PR TITLE
New typography styles

### PR DESCRIPTION
**Pull request by marciovicente**
image

image

### CODE REVIEW

File: scss/dependencies/_mixins.scss [hunk 1]
```diff
@@ -33,7 +33,7 @@
 }
 
 @function pt-to-rem($pt) {
-  @return ($pt * 0.133) * 1rem;
+  @return ($pt * 0.133 * 0.625) * 1rem;
```
Comment (rafaelverger) [inline scss/dependencies/_mixins.scss]: where did you find this calc?

**Comment by rafaelverger** [comment]
I think we should rename `typography-*` by `typo-*`

File: scss/dependencies/_mixins.scss (hunk 1 above)
Comment (marciovicente) [inline scss/dependencies/_mixins.scss]: `1px == 0.625rem` and `1pt == 1.33px`. We need to transform `pt` to `rem`. (Authored by @emersonfarias) :P

**Comment by marciovicente** [comment]
@rafaelverger I think `typo` is weird, once that's a write failure.

File: scss/dependencies/_mixins.scss (hunk 1 above)
Comment (rafaelverger) [inline scss/dependencies/_mixins.scss]: Nice catch... use variables with semantic name to store these "magic numbers"

**Comment by rafaelverger** [comment]
I see.. in fact typo means typography error :/ but keep writing `typography-something` for each node that doesn't have default behaviour is really bad :S

**Comment by marciovicente** [comment]
I agree @rafaelverger, but I prefer `text-` as prefix. What you think about @andersonba @shankarcabus ?

**Comment by andersonba** [comment]
I agree! :clap:

**Comment by marciovicente** [comment]
Ok, I'll do it later, once that's a big impact in others projects. Let's just review this PR context.

**Comment by rafaelverger** [comment]
Nice... it's all good to me

**Pull request merged**